src/test/ui/issue-13483.rs

```diff
@@ -14,9 +14,3 @@ fn main() {
     } else {
     };
 }
-
-fn foo() {
-    if true {
-    } else if { //ERROR: MISSING CONDITIONAL
```

Comment:
@nikomatsakis changed. It doesn't catch the following case, but I feel it is an acceptable tradeoff, given the alternatives:

```rust
if true {
} else if {
    true
}
```